articles/qos.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: ROS 2.0 Quality of Service policies
+abstract:
+  This article describes the approach to provide QoS (Quality of Service) policies for ROS 2.0.
+published: false
+author: '[Esteve Fernandez](https://github.com/esteve)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+With the advent of inexpensive robots using unreliable wireless networks, developers and users need mechanisms to control how traffic is prioritized across network links.
+
+## Background
+
+ROS 1.x uses TCP as the underlying transport, which is unsuitable for lossy networks such as wireless links.
+With ROS 2.0 relying on DDS which uses UDP as its transport, we can give control over the level of reliability a node can expect and act accordingly.
+
+## DDS Quality of Service policies
+
+DDS provides fine-grained control over the Quality of Service (QoS) setting for each of the entities involved in the system.
+Common entities whose behavior can be modified via QoS settings include: Topic, DataReader, DataWriter, Publisher and Subscriber.
+QoS is enforced based on a Request vs Offerent Model, however Publications and Subscriptions will only match if the QoS settings are compatible.
```

Comment:
Unless I'm mistaken, in English I believe this is `Offerer` rather than `Offerent`.